unit_test/batched/Test_Batched_TeamVectorGemm_Real.hpp

```diff
@@ -0,0 +1,80 @@
+#if defined(HAVE_KOKKOSKERNELS_HALFMATH)
+TEST_F( TestCategory, batched_scalar_team_vector_gemm_nt_nt_half_half ) {
```

Comment:
Note: This may run with `::Test::halfScalarType = float` if half precision is available on only the host or device.